frontend/src/AppRoutes/index.tsx

```diff
@@ -1,40 +1,47 @@
-import { ConfigProvider } from 'antd';
+import { ConfigProvider, notification } from 'antd';
 import NotFound from 'components/NotFound';
 import Spinner from 'components/Spinner';
 import AppLayout from 'container/AppLayout';
 import { useThemeConfig } from 'hooks/useDarkMode';
 import history from 'lib/history';
-import React, { Suspense } from 'react';
+import React, { Suspense, useMemo } from 'react';
 import { Route, Router, Switch } from 'react-router-dom';
 
+import { NotificationContext } from './notifications';
 import PrivateRoute from './Private';
 import routes from './routes';
 
 function App(): JSX.Element {
 	const themeConfig = useThemeConfig();
-
+	const [notificationApi, NotificationElement] = notification.useNotification();
+	const notifications = useMemo(() => ({ notifications: notificationApi }), [
+		notificationApi,
+	]);
 	return (
 		<ConfigProvider theme={themeConfig}>
-			<Router history={history}>
-				<PrivateRoute>
-					<AppLayout>
-						<Suspense fallback={<Spinner size="large" tip="Loading..." />}>
-							<Switch>
-								{routes.map(({ path, component, exact }) => (
-									<Route
-										key={`${path}`}
-										exact={exact}
-										path={path}
-										component={component}
-									/>
-								))}
+			<NotificationContext.Provider value={notifications}>
```

Comment:
Also please make the component for this provider in the hook itself and use it as <NotificationProvider>